types/web3/eth/index.d.ts

```diff
@@ -166,12 +166,12 @@ export default interface Eth {
     ): Promise<EncodedTransaction>;
     sendSignedTransaction(
         data: string,
-        cb?: Callback<string>
-    ): PromiEvent<TransactionReceipt>;
+        cb?: Callback<TransactionReceipt | string | { confNumber: number, receipt: TransactionReceipt }>
+    ): PromiEvent<TransactionReceipt | string | { confNumber: number, receipt: TransactionReceipt }>;
```

Comment:
I am not sure how the PromiEvent is exactly typed, but the promise part of this will only resolve with the "hash" if I understand correctly. There are also some events on it that send the `receipt` and the `confirmationNumber` but these are not passed to the resolve function of the promise. (This kind of "rx js" behaviour of this promise is overcomplex I find, but what can we do here?)